config_utilities/include/config_utilities/types/enum.h

```diff
@@ -213,19 +211,13 @@ void enum_field(EnumT& field, const std::string& field_name, const std::map<Enum
   }
   auto& converter = Enum<EnumT>::instance();
   converter.mutex_.lock();
-  const auto backup = std::move(converter.enum_names_);
```

Comment:
Similar to the above, I'd suggest keeping that behavior as is (we thought a while about how to best handle this and it seems that this seemed to be the most flexible option)